### PR TITLE
Dev-6810 Tab Styles

### PR DESCRIPTION
**Pull request by jonhill13**
### Author
- [x] Story Book has been rebuilt (new build artifacts in `/docs`)
- [x] Components have been rebuilt (new build artifacts in `/dist`)

👆These can be done by executing `npm run build` and committing the build artifacts.

- [x] All new components have been exported in `/index.js` `(if applicable)`
- [x] Design Review Complete
- [x] Relevant Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/release_notes_template.md) to document your changes) `(if applicable)`
- [x] Integration Status Updated on relevant documentation page `(if applicable)`
- [x] `index.d.ts` updated with new prop types defined for new components. `(if applicable)`

### Reviewer
- [x] Code review

### CODE REVIEW

File: tests/components/Tabs-test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    ],
+    active: 'name',
+    switchTab: () => {}
+};
+
+const switchTab = (tab) => {
+  defaultProps.active = tab
+};
+
+describe('SearchBar', () => {
```
Comment (MaxwellKendall) [inline tests/components/Tabs-test.js:28]: searchbar?

File: tests/components/Tabs-test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          render(<Tabs {...defaultProps} />);
+          expect(screen.getByText('23')).toBeTruthy();
+        });
+        it('it should show tooltip', () => {
+          render(<Tabs {...defaultProps} />);
+          expect(screen.getByRole('button').classList.contains('tooltip__hover-wrapper')).toBeTruthy();
+        });
+        it('it should not switch tab when tab is disabled', async () => {
+          defaultProps.types[1] = { internal: 'dates', label: 'Dates', disabled: true };
+          console.log(' Default Props : ', defaultProps);
```
Comment (MaxwellKendall) [inline tests/components/Tabs-test.js:55]: perhaps remove this?

File: components/tabs/Tabs.jsx [hunk 3]
```diff
@@ -14,18 +13,23 @@ const propTypes = {
...
+        count: PropTypes.number,
+        disabled: PropTypes.bool,
+        tooltip: PropTypes.element,
+        count: PropTypes.number
     })).isRequired,
     active: PropTypes.string.isRequired,
     switchTab: PropTypes.func.isRequired,
-    tabsClassName: PropTypes.string
+    tabsClassName: PropTypes.string,
+    tablessStyle: PropTypes.bool
```
Comment (MaxwellKendall) [inline components/tabs/Tabs.jsx:24]: Not gonna die on this hill but would prefer this to be an enum. like `tabType` maybe?

**Review by MaxwellKendall — COMMENTED** [review]
Very Nice! Caught a misspelling and a few console logs. Recommend changing the prop name `tablessStyle` to an enum. Thoughts?

File: tests/components/Tabs-test.js (hunk 1 above)
Comment (MaxwellKendall) [reply tests/components/Tabs-test.js:28]: assuming this should say tabs. Also, we dont really need to have nested describe statements. When we execute the test suite jest prints out the test file thats being logged so we dont really need to do this IMO.

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
Remove logs, correct mis-spellings. Recommend renaming prop type.

File: tests/components/Tabs-test.js (hunk 1 above)
Comment (jonhill13) [reply tests/components/Tabs-test.js:28]: agreed glad to see you changed your mind on this

File: .storybook/stories/tabs.stories.mdx [hunk 4]
```diff
@@ -22,38 +22,201 @@ import Tabs from '../../components/tabs/Tabs';
...
 In this example:
 
 ### Other props:
  - `tabsClassName`: A class name for the tabs
+ - `tablessStyle`: A property to show tabs in a tabless style
 
-```javascript
 
-```
+## Tabs with a Mix of Labes, Counts, and Tooltips
```
[the remark this answers is not in the document]
Comment (jonhill13) [reply .storybook/stories/tabs.stories.mdx:41]: thanks either i cannot spell or sometimes my keyboard connection misses things

File: tests/components/Tabs-test.js (hunk 2 above)
Comment (jonhill13) [reply tests/components/Tabs-test.js:55]: perhaps, yes

File: components/tabs/Tabs.jsx (hunk 3 above)
Comment (jonhill13) [reply components/tabs/Tabs.jsx:24]: I think `tablessStyle` is either 0 or 1 or true or false and why it is a boolean. In the future if we have more tab styles we can apply different classnames based on the different styles created by the design team and this `tablessStyle` can become something like `style` or `tabStyle` and can be an enum

**Review by MaxwellKendall — APPROVED** [review]

**Pull request merged**